synapse/models/language.py

```diff
@@ -0,0 +1,47 @@
+
+def getDataModel():
+    return {
+
+        'prefix':'lang',
+
+        'types':(
+            #('lang:code',{'subof','enum',...
+            ('lang:idiom',{'subof':'str:txt','doc':'A subcultural idiom'}),
+            ('lang:trans',{'subof':'str:txt','doc':'Raw text with a documented translation'}),
+        ),
+
+        'forms':(
+
+            ('lang:idiom',{},(
+                ('url',         {'ptype':'inet:url','doc':'Authoritative URL for the idiom'}),
+                ('desc:en',     {'ptype':'str:txt','doc':'English description'}),
```

Comment:
since it's a "reasonably sized" humon text description, i'd say dragging in byte blobs here would be more pain than it's worth...